src/components/ErrorBoundaryContainer.tsx

```diff
@@ -27,7 +27,6 @@ const Toggle = ({ children, expand, title }: { children?: any; expand?: boolean;
         >
           <g>
             <path
-              className='glyph-fg triangle'
```

Comment:
Where does the `ErrorBoundaryContainer` toggle get styled now?

Reply:
I believe it was never styled before, I inspected it, and the classes for glyph-fg and triangle weren't applied, since all the styles where where descendant selectors.

Reply:
Okay, thanks! That's very strange, as I did a git blame and it never seemed to be connected even when it was first created 🤷‍♀️

For dead code that is removed, if you're able to call that out in the PR for me that would be great. Will save me time on the review.

Reply:
Sounds good! I'll let you know next time.